rails/spec/requests/api/public_stories_spec.rb

```diff
@@ -83,6 +86,40 @@ def json_response
       )
     end
 
+    # Story:
+    # - place 1
+    # - type of place: being deleted
```

Comment:
I would take `type of place` out of the comments here and in `story_5` since it's a different field and not relevant to the tests.